meshpy/mesh_creation_functions/beam_basic_geometry.py

```diff
@@ -516,14 +516,18 @@ def create_beam_mesh_helix(
         bottom plane of helix.
     start_point: np.array, list
         Start point of the helix. Defines the radius.
-    twist_angle: float
-        Twist angle / Pitch of helix between 0 and 2pi.
+    helix_angle: float
+        Angle of the helix (synonyms in literature: twist angle or pitch
+        angle). Exactly two arguments of [helix_angle, height_helix, turns]
+        must be provided.
     height_helix: float
-        Height of helix. Mutually exclusive with number of turns.
+        Height of helix. Exactly two arguments of [helix_angle, height_helix,
+        turns] must be provided.
     turns: float
-        Number of turns. Mutually exclusive with height of helix.
+        Number of turns. Exactly two arguments of [helix_angle, height_helix,
+        turns] must be provided.
```

Comment:
I would just put `Exactly two arguments of [helix_angle, height_helix, turns] must be provided.` in the main description and not in each argument.

Reply:
Thanks for the review and good idea! I've just pushed changes to improve the docstring and ease of the helix function.